package.json

```diff
@@ -1,11 +1,17 @@
 {
-  "name": "zjs",
+  "name": "zwave-js-server",
   "version": "1.0.0",
-  "description": "",
-  "main": "build/index.js",
+  "description": "Full access to zwave-js driver throught Websockets",
+  "main": "lib/index.js",
+  "bin": {
+    "zwave-server": "dist/bin/server.js",
+    "zwave-client": "dist/bin/client.js"
+  },
   "scripts": {
     "lint": "eslint --ext .ts .",
-    "lint:fix": "eslint --fix --ext .ts ."
+    "lint:fix": "eslint --fix --ext .ts .",
+    "build": "tsc -p .",
+    "local": "npm i -g && zwave-server"
```

Comment:
We should not install things globally. Also, the zwave server needs to have CLI args passed.

Reply:
This was just for local testing :)

Reply:
With this PR the module gets installed correctly. A user will run `npm install @zwave-js/server` and then can start the server with `zwave-server /dev/ttyACM0` or the client with `zwave-client`

Reply:
Yeah that sounds good. But then let's drop the local global install. It's not needed.